src/UI/NavigationContainer.elm

```diff
@@ -235,3 +291,40 @@ menu applier { menuExpanded } =
 lazyMap : (a -> b) -> (c -> a) -> (c -> b)
 lazyMap applier original =
     \whatever -> applier (original whatever)
+
+
+dialogView : RenderConfig -> Dialog msg -> Element msg
```

Comment:
Maybe it would be better to move this to another module? Maybe `Internal.Dialog`?

Reply:
Yep, agreed!